website/tests/a11y-helper.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+export const globalAxeOptions = {
+  rules: {
+    'color-contrast': { enabled: false },
+    list: { enabled: false },
```

Comment:
Same for this rule. No rule should be turned off globally but instead per instance that's needed.